Healex.HL7v2Anonymizer/Services/Anonymizer.cs

```diff
@@ -42,9 +46,45 @@ public class Anonymizer {
 
         #region Auxiliar Methods
 
+        public List<Replacement> ExpandReplacementRules(Segment segment, Replacement[] originalRules)
```

Comment:
Please consider pointing out, that only rules for repeated fields are expanded here. You could included it in the name like `ExpandReplacementRulesForRepeatedFields` or add a comment.

Reply:
I like that. Done